ledger/src/blocktree.rs

```diff
@@ -1129,6 +1131,24 @@ impl Blocktree {
         }
     }
 
+    // The `get_block_time` method is not fully implemented (depends on validator timestamp
+    // transactions). It currently returns Some(`slot` * DEFAULT_MS_PER_SLOT) offset from 0 for all
+    // transactions, and None for any values that would overflow any step.
+    pub fn get_block_time(&self, slot: Slot) -> Option<UnixTimestamp> {
+        let (offset_millis, overflow) = slot.overflowing_mul(DEFAULT_MS_PER_SLOT);
```

Comment:
Any way we can get to `bank.ticks_per_slot` here instead of using DEFAULT_MS_PER_SLOT?

Reply:
Oh, duh, yes

Reply:
@mvines , should this still depend on DEFAULT_TICKS_PER_SECOND? Or try to get the denominator from genesis somehow (could reverse calculate it from bank.slots_per_year... or maybe I'm missing something obvious and better)

Reply:
I don't see anything better than `bank.slots_per_year` right now